models/requests.js

```diff
@@ -0,0 +1,59 @@
+module.exports = function(sequelize, DataTypes) {
+    var Request = sequelize.define("Request", {
+      // recipient_id: {
+      //   type: DataTypes.INTEGER,
+      //   primaryKey:true,
+      //   allowNull: false,
+      //   autoIncrement: true,
+      //   unique: true
+      // },
+      // services_id: {
+      //   type: DataTypes.INTEGER,
+      //   primaryKey:true,
+      //   allowNull: false,
+      //   autoIncrement: true,
+      //   unique: true
+      // },
+      finalized: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false
```

Comment:
Let's add an `allowNull: false` here. I've heard if we don't need the NULL value, disallowing them will help improve the performance of the database.